src/file.rs

```diff
@@ -36,7 +36,7 @@ pub struct FileInfo {
 
 impl FileInfo {
     /// Returns the underlying path.
-    pub fn path(&self) -> &Path {
+    pub fn path(&self) -> &Vec<PathBuf> {
```

Comment:
Using `Path` here would break the compiler. Since we already type is as `PathBuf` above I saw no reasons to fight the compiler here.